### PR TITLE
Pic 815 offender assessment

### PR DESCRIPTION
**Pull request by katie-cawthorne**
…issue

- Added 'What to include' summary with hint text
- Added details text area for each issue
- Removed radio buttons and associated guidance text
- Removed evidence section

### CODE REVIEW

File: app/views/oralReport/page6.scala.html [hunk 1]
```diff
@@ -11,93 +11,52 @@
...
-                @checkbox(reportForm("issueHealth"), '_label -> "Physical & mental health")
-                @checkbox(reportForm("issueBehaviour"), '_label -> "Thinking & behaviour")
-                @checkbox(reportForm("issueMaturity"), '_label -> "Maturity")
+                @checkboxDetailsField(reportForm, "issueAccommodation", "Accommodation", "Provide a brief assessment of accommodation")
+                @checkboxDetailsField(reportForm, "issueEmployment", "Employment, training and education", "Provide a brief assessment of employment, training and education")
+                @checkboxDetailsField(reportForm, "issueFinance", "Finances", "Provide a brief assessment of finances")
+                @checkboxDetailsField(reportForm, "issueRelationships", "Relationships", "Provide a brief assessment of relationships")
+                @checkboxDetailsField(reportForm, "issueSubstanceMisuse", "Substance misuse", "Provide a brief assessment of substance misuse")
+                @checkboxDetailsField(reportForm, "issueHealth", "Physical & mental health", "Provide a brief assessment of physical & mental health")
```
Comment (theDustRoom) [inline app/views/oralReport/page6.scala.html:54]: "Experience of trauma" and "Caring Responsibilities" are on the original report template but have been omitted on the Word document.

I've checked with Alastair and they should be on here, can you include them in the same way here?

File: app/views/oralReport/page6.scala.html [hunk 2]
```diff
@@ -11,93 +11,52 @@
...
-                        "no" -> "No"
-                    ),
-                    Symbol("_inline") -> "true"
-                )
-            </fieldset>
-        </div>
-
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-
-        @textarea(reportForm("assessmentEvidence"), '_label -> "Evidence for assessment")
```
Comment (theDustRoom) [inline app/views/oralReport/page6.scala.html:102]: Does this also need removing from OralReportData and the hidden inputs file?

**Review by theDustRoom — CHANGES_REQUESTED** [review]
Checked with Alastair on the report form. changes inline

File: app/views/oralReport/page6.scala.html (hunk 1 above)
Comment (katie-cawthorne) [reply app/views/oralReport/page6.scala.html:54]: Yeah I was just following the word doc and removed them as they weren't there.

I'll add them back in

File: app/views/oralReport/page6.scala.html (hunk 2 above)
Comment (katie-cawthorne) [reply app/views/oralReport/page6.scala.html:102]: I thought I'd removed it but I'll have a look

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**